kupyna512/src/sub_units/t_xor_plus/tests/test_individual_layers.rs

```diff
@@ -0,0 +1,146 @@
+use crate::sub_units::t_xor_plus::*;
+
+#[test]
+fn test_add_constant_xor() {
+    let input = [
+        0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08, 0x09, 0x0A, 0x0B, 0x0C, 0x0D, 0x0E,
```

Comment:
You can use the [`hex-literal`](https://docs.rs/hex-literal/latest/hex_literal/) crate for such constants (add it to `dev-dependencies`).